### PR TITLE
Add Seq2SeqORTTrainer and translation example

### PR DESCRIPTION
**Pull request by JingyaHuang**
# What does this PR do?

* Add seq2seq trainer for ONNX Runtime training.
* Add a translation example.
* Add some benchmark results in Readmes.

### CODE REVIEW

File: examples/onnxruntime/training/text-classification/README.md [hunk 1]
```diff
@@ -39,14 +39,29 @@ python run_glue.py \
     --output_dir /tmp/ort-bert-sst2/
 ```
 
+### Performance
+
+We get the following results for [roberta-base](https://huggingface.co/roberta-base) and [roberta-large](https://huggingface.co/roberta-large) 
+on sst2 dataset under PyTorch and ONNX Runtime backends. (a single Nvidia A100 was used):
+
+| Run configuration            | FP16 | Runtime | Train samples(/s) |
```
Comment (echarlaix) [inline examples/onnxruntime/training/text-classification/README.md:47]: Unit of measurement missing for runtime

File: examples/onnxruntime/training/text-classification/README.md [hunk 2]
```diff
@@ -39,14 +39,29 @@ python run_glue.py \
...
+| ---------------------------- | ---- | ------- | ----------------- |
+| roberta-base / PyTorch       | TRUE | 752.3   | 268.589           |
```
Comment (echarlaix) [inline examples/onnxruntime/training/text-classification/README.md:49]: If FP16 always set to True, we can remove this column and note this information in the introduction sentence before the table. We can even create an other column to separate the informations displayed in the column "Run configuration"

File: examples/onnxruntime/training/token-classification/README.md [hunk 3]
```diff
@@ -32,13 +32,30 @@ The following example fine-tunes a BERT on the sst-2 task.
...
     --dataset_name conll2003 \
     --do_train \
     --do_eval \
     --output_dir /tmp/ort_bert_conll2003/
 ```
 
+### Performance
+
+We get the following results with the previous commands under PyTorch and ONNX Runtime backends. (a single Nvidia A100 was
+used to run 7 epochs):
```
Comment (echarlaix) [inline examples/onnxruntime/training/token-classification/README.md:45]: You could remove the parenthesis

**Review by echarlaix — COMMENTED** [review]
Great work @JingyaHuang !! I left a couple of comments but nothing important.

**Review by echarlaix — APPROVED** [review]

File: examples/onnxruntime/training/translation/README.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+### Performance
+
+We get the following results with the previous commands under PyTorch and ONNX Runtime backends. (a single Nvidia A100 was
+used):
+
+| Backend      | FP16 | Runtime | Train samples(/s) | 
+| ------------ | ---- | ------- | ----------------- | 
+| PyTorch      | TRUE | 2038.8  | 44.143            | 
+| ONNX Runtime | TRUE | 1536.7  | 58.566            |
```
Comment (echarlaix) [inline examples/onnxruntime/training/translation/README.md:63]: You can also reduce the number of significant digits for clarity

File: tests/onnxruntime/test_onnxruntime_train.py [hunk 5]
```diff
@@ -120,6 +121,119 @@ def compute_metrics(eval_pred):
...
+                            tokenizer=tokenizer,
+                            data_collator=data_collator,
+                            feature="seq2seq-lm",
+                        )
+
+                        train_result = trainer.train()
+                        trainer.save_model()
+                        train_metrics = train_result.metrics
+                        # ort_eval_metrics = trainer.evaluate()
+                        # ort_prediction = trainer.predict(test_dataset)
```
Comment (echarlaix) [inline tests/onnxruntime/test_onnxruntime_train.py:232]: forgotten comments

File: examples/onnxruntime/training/translation/README.md (hunk 4 above)
Comment (JingyaHuang) [reply examples/onnxruntime/training/translation/README.md:63]: Yeah, will do!

File: examples/onnxruntime/training/text-classification/README.md (hunk 1 above)
Comment (JingyaHuang) [reply examples/onnxruntime/training/text-classification/README.md:47]: Roger that!

File: examples/onnxruntime/training/text-classification/README.md (hunk 2 above)
Comment (JingyaHuang) [reply examples/onnxruntime/training/text-classification/README.md:49]: Good point!

**Comment by JingyaHuang** [comment]
Thanks for the review, I will merge the branch. 
Another PR will be created to deal with the broadcast error, and the PR for the `ORTTrainer` notebook will be available now thanks to the change in this PR.

**Pull request merged**